GAE/src/org/waterforpeople/mapping/app/web/RawDataRestServlet.java

```diff
@@ -426,14 +425,16 @@ private SurveyInstance createInstance(RawDataImportRequest importReq) {
         SurveyInstance inst = new SurveyInstance();
         inst.setUserID(1L);
         inst.setSurveyId(importReq.getSurveyId());
-        inst.setCollectionDate(importReq.getCollectionDate() != null ? importReq
-                .getCollectionDate() : new Date());
+        inst.setCollectionDate(importReq.getCollectionDate() != null 
+        		? importReq.getCollectionDate()
+        		: new Date());
         inst.setDeviceIdentifier("IMPORTER");
         inst.setUuid(UUID.randomUUID().toString());
         inst.setSurveyedLocaleId(importReq.getSurveyedLocaleId());
         inst.setUuid(UUID.randomUUID().toString());
         inst.setSubmitterName(importReq.getSubmitter());
         inst.setSurveyalTime(importReq.getSurveyDuration());
+        inst.setFormVersion(importReq.getFormVersion());
```

Comment:
Does this mean its possible to (re)set the form version when (re)importing data?

Reply:
Only when importing new data. It will be ignored when reimporting.

Reply:
Jana said to handle it just like SurveyalTime, so I did.